x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/pki/PkiRealm.java

```diff
@@ -317,6 +317,17 @@ public void expireAll() {
         }
     }
 
+    @Override
+    public void usageStats(ActionListener<Map<String, Object>> listener) {
+        super.usageStats(ActionListener.wrap(stats -> {
+            stats.put("has_truststore", trustManager != null);
+            stats.put("has_delegated_realms", delegatedRealms != null);
```

Comment:
`delegatedRealms` should never be null (assumig the `initialize` method has been called).
You want `delegatedRealms.hasDelegation()`

Reply:
Great catch. Thank you!